### PR TITLE
Introduce new attribute for links to access.redhat.com

### PR DESCRIPTION
**Pull request by maximiliankolb**
Hi!
This PR introduces a new attribute `AccessRedHatComVersion` used in links to `access.redhat.com`.
This allows the upstream documentation to use its own version number and still link to the latest Satellite documentation where appropriate.

Example where it was showing the Satellite version in the Foreman docs: https://docs.theforeman.org/nightly/Upgrading_and_Updating/index-foreman-el.html#updating_satellite_server_to_next_minor_version:  `# foreman-maintain upgrade check --target-version 6.9.z`

### CODE REVIEW

**Comment by melcorr** [comment]
Can you rebase so that I can test!

**Comment by melcorr** [comment]
In general, I agree with this suggestion, so if you could update the readme as part of this PR, I would appreciate it. Then I will test and merge.

**Comment by maximiliankolb** [comment]
> In general, I agree with this suggestion, so if you could update the readme as part of this PR, I would appreciate it. Then I will test and merge.

I have rebased to master & adjusted the `README`.  Is there any other place that needs to be updated?

**Comment by maximiliankolb** [comment]
Rebased to master.

File: guides/common/attributes-base.adoc [hunk 1]
```diff
@@ -85,6 +85,8 @@
 :ProjectX: Foreman
 // FIXME: This should be: :ProjectXY: Foreman{nbsp}{ProjectVersion}
 :ProjectXY: Foreman{nbsp}1.22
+:ProductVersion: 2.4
```
Comment (melcorr) [inline guides/common/attributes-base.adoc:88]: should this not be 2.5 now?

File: guides/common/attributes-base.adoc [hunk 2]
```diff
@@ -85,6 +85,8 @@
...
+:ProductVersionPrevious: 2.3
```
Comment (melcorr) [inline guides/common/attributes-base.adoc:89]: and 2.4 here?

File: guides/doc-Administering_Red_Hat_Satellite/topics/Maintenance.adoc [hunk 3]
```diff
@@ -160,7 +160,7 @@ If you want to check for updates using `yum`, enter the command to install and u
 # {foreman-maintain} packages lock
 ----
 Updating packages individually can lead to package inconsistencies in {Project} or {SmartProxy}.
-For more information about updating packages in {Project}, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProductVersion}/html/upgrading_and_updating_red_hat_satellite/updating_satellite_server_capsule_server_and_content_hosts#updating_satellite_server_to_next_minor_version[Updating {ProjectServer}].
+For more information about updating packages in {Project}, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html/upgrading_and_updating_red_hat_satellite/updating_satellite_server_capsule_server_and_content_hosts#updating_satellite_server_to_next_minor_version[Updating {ProjectServer}].
```
Comment (melcorr) [inline guides/doc-Administering_Red_Hat_Satellite/topics/Maintenance.adoc:163]: We now have this guide upstream so we should be using the correct attribute for it.

**Review by melcorr — CHANGES_REQUESTED** [review]
Few small comments, but that is all. 
Happy to merge this.

File: guides/doc-Administering_Red_Hat_Satellite/topics/Maintenance.adoc (hunk 3 above)
Comment (melcorr) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Maintenance.adoc:163]: :UpgradingDocURL: {BaseURL}Upgrading_and_Updating/index-foreman-el.html# etc in attributes-base

File: guides/common/attributes-base.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/attributes-base.adoc:88]: Fixed.

File: guides/common/attributes-base.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/attributes-base.adoc:89]: Fixed.

**Comment by melcorr** [comment]
@maximiliankolb it says there are conflicts. Can you take a look?

**Comment by maximiliankolb** [comment]
> @maximiliankolb it says there are conflicts. Can you take a look?

Working on it!

File: guides/doc-Administering_Red_Hat_Satellite/topics/Maintenance.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Maintenance.adoc]: Fixed. Can you please re-review this?

**Comment by maximiliankolb** [comment]
Rebased to master.

**Review by melcorr — APPROVED** [review]
Thank you!

**Pull request merged**